### PR TITLE
check for last read at on missed yesterday

### PR DESCRIPTION
**Pull request by robguthrie**

### CODE REVIEW

**Comment by robguthrie** [comment]
@gdpelican  does this look ok to you?

File: app/views/user_mailer/missed_yesterday/_discussion.html.haml [hunk 1]
```diff
@@ -13,7 +13,8 @@
     = render 'thread_mailer/motion', motion: motion
 
   .activity-feed
-    - discussion.items.where('created_at > ?', @time_start).order('created_at').each do |item|
+    - last_read_at = DiscussionReader.for(user: @user, discussion: discussion).last_read_at
```
Comment (gdpelican) [inline app/views/user_mailer/missed_yesterday/_discussion.html.haml]: I know we're in a delayed job here, but I'm a little worried about running that query for every discussion in every group in every email we send out.

Maybe we could use DiscussionReaderCache here somehow?

**Comment by robguthrie** [comment]
Thanks for the great feedback @gdpelican . Implemented

**Comment by gdpelican** [comment]
Yeah, that looks great.

**Pull request merged**